lib/index.js

```diff
@@ -193,7 +193,7 @@ function gitUrlParse(url) {
         urlInfo.organization = urlInfo.owner;
     }
 
-    const bitbucket = /(projects|users)\/(.*?)\/repos\/(.*?)\/(raw|browse)\/(.*?)$/
+    const bitbucket = /(projects|users)\/(.*?)\/repos\/(.*?)\/(raw|browse)\/?(.*?)$/
```

Comment:
```suggestion
    const bitbucket = /(projects|users)\/(.*?)\/repos\/(.*?)\/(raw|browse)(?:\/(.*?))$/
```

This is more in line with what you intended right? Otherwise it might match `/browsebbbbbbb`

Reply:
Awesome! There's a lot of subtlety in regex, so I added two more tests.